docs/source/sample_datasets.rst

```diff
@@ -1,44 +1,53 @@
++-----------------------------------------------------+-------------------------------------------------------------------+-----------------+---------------------+--------+
+| `Electron Bessel beam diffraction`_                 | Scanning convergent beam electron diffraction with ring-shaped    | Stack of DM3    | 3D                  | 2.6 GB |
+| :cite:`giulio_guzzinati_2019_2566137,Guzzinati2019` | aperture and overlapping diffraction orders.                      | (currently only |                     |        |
+|                                                     |                                                                   | scripting)      |                     |        |
++-----------------------------------------------------+-------------------------------------------------------------------+-----------------+---------------------+--------+
 
 Creating random data
 ~~~~~~~~~~~~~~~~~~~~~~~
-Random data can be generated in the following way. It should be kept in mind that the data generated in this way can only be used for simple testing as it has no physical significance.
+
+Random data can be generated in the following way. It should be kept in mind
+that the data generated in this way can only be used for simple testing as it
+has no physical significance.
 
 .. code-block:: python
       
     import numpy as np
     real_data = np.random.randn(16, 16, 16, 16).astype("float32")
     real_data.tofile("/tmp/real_raw_file.raw")
 
-Now you can load the data through the `Python API`_ in the following way
+Now you can load the data through the :ref:`api documentation` in the following way:
 
 .. code-block:: python
     
     from libertem.api import Context
     ctx = Context()
-    ds = ctx.load("raw", path="/tmp/something.raw", scan_size=(16, 16), dtype="float32", detector_size=(16, 16))
+    ds = ctx.load("raw", path="/tmp/real_raw_file.raw", scan_size=(16, 16), dtype="float32", detector_size=(16, 16))
     
-Alternatively, you can enter the parameters (scan_size, dtype, detector_size) directly into the load dialog of the GUI. 
-
-.. _link1: https://zenodo.org/record/3592520/files/calibrationData_bullseyeProbe.h5?download=1
-.. _link2: https://zenodo.org/record/3592520/files/calibrationData_circularProbe.h5?download=1
-.. _link3: https://zenodo.org/record/2566137/files/experimental_data.7z?download=1
-.. _Bullseye probe: https://zenodo.org/record/3592520#.XmdNN3DhXIU
-.. _Circular probe: https://zenodo.org/record/3592520#.XmdNN3DhXIU  
-.. _Electron Bessel beam diffraction pattern: https://zenodo.org/record/2566137#.XmdNQnDhXIU
-.. _Python API: https://libertem.github.io/LiberTEM/api.html
\ No newline at end of file
+Alternatively, you can enter the parameters (scan_size, dtype, detector_size)
+directly into the load dialog of the GUI.
+
+.. _`Bullseye and circular probe diffraction`: https://zenodo.org/record/3592520
+.. _`Electron Bessel beam diffraction`: https://zenodo.org/record/2566137
```

Comment:
```suggestion
.. _`Electron Bessel beam diffraction`: https://zenodo.org/record/2566137
.. bibliography:: libertem-references.bib
```

Reply:
So did you reproduce the error on your system and confirm that this fixes it?

Reply:
On my system I get an error message when I have two bibliographies.

Reply:
> So did you reproduce the error on your system and confirm that this fixes it?

No I haven't confirmed that. I saw [this](https://build-me-the-docs-please.readthedocs.io/en/latest/Using_Sphinx/UsingBibTeXCitationsInSphinx.html#example) in the Sphinx documentation and suggested the changes.



> On my system I get an error message when I have two bibliographies.

How did you test this in your system?

Reply:
`tox -e docs-check` https://libertem.github.io/LiberTEM/contributing.html#building-the-documentation